pkg/fakerp/vault.go

```diff
@@ -75,12 +75,39 @@ func writeTLSCertsToVault(ctx context.Context, kvc azureclient.KeyVaultClient, c
 	return nil
 }
 
-func createVault(ctx context.Context, vc azureclient.VaultMgmtClient, appObjectID, tenantID, resourceGroup, location, vaultName string) (string, error) {
+func createVault(ctx context.Context, vc azureclient.VaultMgmtClient, writeObjectID, readObjectID, tenantID, resourceGroup, location, vaultName string) (string, error) {
 	tID, err := uuid.FromString(tenantID)
 	if err != nil {
 		return "", err
 	}
 
+	aplist := []mgmtkeyvault.AccessPolicyEntry{
+		{
+			TenantID: &tID,
+			ObjectID: &writeObjectID,
+			Permissions: &mgmtkeyvault.Permissions{
+				Certificates: &[]mgmtkeyvault.CertificatePermissions{
+					mgmtkeyvault.Import,
+				},
+				Secrets: &[]mgmtkeyvault.SecretPermissions{
+					mgmtkeyvault.SecretPermissionsGet,
```

Comment:
please add a comment on why this line is here

Reply:
ok